src/p5.dance.interpreted.js

```diff
@@ -121,13 +125,27 @@ function atTimestamp(timestamp, unit, event) {
 }
 
 function everySeconds(n, unit, event) {
```

Comment:
Would  be great to start getting some param/return tags on these functions so that they can be more easily understood going forward